migrations/20210917111754-create-user.js

```diff
@@ -0,0 +1,41 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable("Users", {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      first_name: {
+        type: Sequelize.STRING,
```

Comment:
please define not nullable fields here too and add an empty boiler plate for update, create and delete hooks